bin/deploy_on_minishift

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+project=$PROJECT
+if [ -z "$project" ];
+then
+	project=myproject
+fi
```

Comment:
Can we use something like this here?

```bash
 project="${PROJECT:-myproject}"
```

Reply:
I'm sure we can, bash isn't my first, second, or third language :)